ordinary_data/views/schematic/vw_pipe_schema.sql

```diff
@@ -17,7 +17,6 @@ CREATE OR REPLACE VIEW qwat_od.vw_pipe_schema_visibleitems AS
 		pipe.id,
 		pipe.fk_parent,
 		pipe.fk_material,
-		pipe._length2d,
```

Comment:
you need to recompute the length here with `ST_Length(geometry) AS _length2d`

otherwise we won't have such info since we only keep geometry_alt2